### DISCUSSION TITLE
메인페이지 "스크롤을 내려주세요" 애니메이션 아이디어

### DISCUSSION THREAD

**Discussion by not-using** [jiphyeonjeon-42/frontend#534] [General]
최근 몇몇 분한테 메인페이지에 배너 하단에 컨텐츠가 있는 줄 몰랐다는 피드백을 받았습니다.

스크롤을 내려달라는 문구가 있긴 하지만 눈에 띄지 않아서 그런 거 같아요 
좀 더 눈에 띄도록 애니메이션을 추가하려 하는데 어떤 애니메이션이 좋을까요?

1. 마우스 스크롤
![Aug-14-2023 14-43-13](https://github.com/jiphyeonjeon-42/frontend/assets/74622889/ae0aa61b-dd98-44cb-aacc-a113850af82a)

2. 문구 수직 흔들림
![Aug-14-2023 14-49-30](https://github.com/jiphyeonjeon-42/frontend/assets/74622889/4f914387-0560-4dde-89cc-f02acfeda789)

3. 문구 수평 흔들림
![Aug-14-2023 14-50-54](https://github.com/jiphyeonjeon-42/frontend/assets/74622889/bee90888-1e2b-4d61-8558-55e38638d679)

4. 회전 흔들림
![Aug-14-2023 14-51-33](https://github.com/jiphyeonjeon-42/frontend/assets/74622889/c4f01420-eb38-4e8c-ba33-5df65193c42b)

5. 확대 축소
![Aug-14-2023 14-52-24](https://github.com/jiphyeonjeon-42/frontend/assets/74622889/011f0b1d-fc01-4109-b48f-e861a09e811c)

6. 기타 요란한 애니메이션
![Aug-14-2023 14-48-24](https://github.com/jiphyeonjeon-42/frontend/assets/74622889/38a80063-ef47-4abb-b1df-ee85a3613df3)

#533

**Comment by nyj001012** [comment]
1, 2번 마음에 듭니다 :)

**Reply by chanhihi** [reply]
저도 1,2번 😄

**Reply by mink97** [reply]
저도 2번이요(글씨는 안움직이는 것 찬성합니다!)

**Reply by jimin52** [reply]
2번이요

**Reply by Jake1152** [reply]
1번이요
